src/coldcard.js

```diff
@@ -325,60 +316,63 @@ export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
    * @param {array<object>} options.inputs - inputs for the transaction
    * @param {array<object>} options.outputs - outputs for the transaction
    * @param {array<string>} options.bip32Paths - BIP32 paths
-   * @param {object} options.psbt - optional PSBT
+   * @param {object} options.psbt - PSBT of the transaction to sign
    */
   constructor({network, inputs, outputs, bip32Paths, psbt}) {
     super();
-    this.network = network;
-    this.inputs = inputs;
-    this.outputs = outputs;
-    this.bip32Paths = bip32Paths;
-    this.psbt = psbt;
+    if (psbt) {
```

Comment:
what about just 
```
if (!psbt) {
 throw new Error('...');
}
...
```
makes it more readable imo, especially if this constructor grows (e.g. to do some validation)

Reply:
I agree with you, but eslint does not. 😬

Reply:
Oh interesting -- if you do 

```
if (!psbt) {
 throw new Error('...');
}
...
```

That's okay but if you do

```
if (!psbt) {
 throw new Error('...');
} else {
  ...
}
```

It shows as an eslint error. I just took it to mean can't do any negated condition, but apparently that's incorrect.

Reply:
Oh yeah, that's probably like with `return`s. I forget what the rule is but I think I saw it in our other linting PR. Basically the else is unnecessary if you throw or return since if the condition is true, it'll never get past anyway and there's no need for a guard statement.